### DISCUSSION TITLE
Changing App\Orchid namespace for packages

### DISCUSSION THREAD

**Discussion by bilogic** [orchidsoftware/platform#1620] [Q&A]
I need a different namespace for packages. Is it possible to specify a different namespace for the artisan commands?

```
php artisan orchid:screen EmailSenderScreen
php artisan orchid:table PostListLayout
```

**Comment by kdv88** [comment]
You can use something like

`php artisan orchid:screen App\\Orchid\\Screens\\Admin\\Posts\\PostListLayout`

**Reply by bilogic** [reply]
Oh, that works? If I wasn't clear enough, what I meant was something like `Bilogic\CRM\SendEmailScreen` in a composer package. Let me try it out later, thanks!

**Reply by bilogic** [reply]
@kdv88 it doesn't work.

`php artisan orchid:screen /Bilogic/CRM/SendEmailScreen` => `namespace App\Orchid\Screens\Bilogic\CRM;`

I'm expecting `namespace Bilogic\CRM;`
